plugins/modules/efs.py

```diff
@@ -502,6 +527,30 @@ def update_lifecycle_policy(self, name, transition_to_ia):
                 changed = True
         return changed
 
+    def change_backup_policy(self, name, status):
+        """
+        Enable/disable automatic backup policy of the FileSystem.
+        """
+        changed = False
+        state = self.get_file_system_state(name)
+        if state in [self.STATE_AVAILABLE, self.STATE_CREATING]:
+            wait_for(
+                lambda: self.get_file_system_state(name),
+                self.STATE_AVAILABLE,
+                self.wait_timeout
+            )
+            fs_id = self.get_file_system_id(name)
+            current_backup_policy = self.get_backup_policy(fs_id)
+            params = dict()
```

Comment:
The code is valide, but `params` in general is already used for the module parameters. Can you use a different name for your variable?

Reply:
Sure